pkg/oauth/apiserver/apiserver.go

```diff
@@ -3,24 +3,39 @@ package apiserver
 import (
 	"fmt"
 	"sync"
+	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apiserver/pkg/registry/rest"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
 
 	oauthapiv1 "github.com/openshift/api/oauth/v1"
-	oauthclient "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	oauthclients "github.com/openshift/client-go/oauth/clientset/versioned"
+	oauthinformer "github.com/openshift/client-go/oauth/informers/externalversions"
 	routeclient "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
+	userclient "github.com/openshift/client-go/user/clientset/versioned"
+	userinformer "github.com/openshift/client-go/user/informers/externalversions"
+	bootstrap "github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
 	"github.com/openshift/library-go/pkg/oauth/oauthserviceaccountclient"
 
 	accesstokenetcd "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/oauthaccesstoken/etcd"
 	authorizetokenetcd "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/oauthauthorizetoken/etcd"
 	clientetcd "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/oauthclient/etcd"
 	clientauthetcd "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/oauthclientauthorization/etcd"
+	tokenreviews "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/tokenreviews"
 	useroauthaccesstokensdelegate "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver/registry/useroauthaccesstokens/delegate"
 	"github.com/openshift/oauth-apiserver/pkg/serverscheme"
+	"github.com/openshift/oauth-apiserver/pkg/tokenreviews/tokenvalidation"
+	"github.com/openshift/oauth-apiserver/pkg/tokenreviews/tokenvalidation/usercache"
+	tokenvalidators "github.com/openshift/oauth-apiserver/pkg/tokenreviews/tokenvalidation/validators"
+)
+
+const (
+	defaultInformerResyncPeriod     = 10 * time.Minute
+	minimumInactivityTimeoutSeconds = 300
```

Comment:
these are the old values from kube-apiserver I assume.

Reply:
Yes:
https://github.com/openshift/kubernetes/blob/4b40bb4792d0b183fd02636660c55faac08a8735/cmd/kube-controller-manager/app/patch_informers_openshift.go#L171
https://github.com/openshift/kubernetes/blob/4b40bb4792d0b183fd02636660c55faac08a8735/pkg/kubeapiserver/authenticator/patch_authenticator.go#L65